docs/explanations/certificates.md

```diff
@@ -0,0 +1,44 @@
+---
+sidebar_label: Certificates rotation
+sidebar_position: 21
+title: Certificates rotation 
+description: A description of how Kubewarden controller manages it certificates
+keywords: [ kubewarden, certificate, controller, reconciliation ]
+doc-persona: [kubewarden-operator]
+doc-type: [explanation]
+doc-topic: [explanations, certificates]
+---
+
+<head>
+  <link rel="canonical" href="https://docs.kubewarden.io/explanations/certificates"/>
+</head>
+
+
+Since Kubewarden v1.17.0, the cert-manager dependency was removed. The
+controller is able to manage all the certificates used by all the components. This means
+that the controller has a new reconciliation loop that ensures the certificates
+are always up-to-date and the webhook configuration is correct.
+
+The first certificates generation is done by the Helm chart installation. It
+will generate the first root CA with 10 years of expiration. And the controller
```

Comment:
maybe we can rephrase without "first" as it was used in the previous sentence.
"It will generate a root CA"